user_badges.module

```diff
@@ -37,10 +37,15 @@ function user_badges_theme() {
  */
 function user_badges_user_presave(Drupal\Core\Entity\EntityInterface $entity) {
```

Comment:
This function have one issue: Steps to reproduce it
1. create roles _Test1_ and _Test2_
2. Create Badge Say **Contributor** and assign both roles to this badge.
3. Edit user profile and assign both roles to current user.
4. At that time user should get only one badge and that is **Contributor**. However because of logic in this function it is assigning same badge twice as we have two roles pointing to same badge.